src/ResourceManager/IotCentral/Commands.IotCentral/help/Remove-AzureRmIotCentralApp.md

```diff
@@ -0,0 +1,198 @@
+---
+external help file: Microsoft.Azure.Commands.IotCentral.dll-Help.xml
+Module Name: AzureRM.IotCentral
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.iotcentral/remove-azurermiotcentralapp
+schema: 2.0.0
+---
+
+# Remove-AzureRmIotCentralApp
+
+## SYNOPSIS
+Deletes an IoT Central Application.
+
+## SYNTAX
+
+### ResourceIdParameterSet (Default)
+```
+Remove-AzureRmIotCentralApp [-PassThru] -ResourceId <String> [-AsJob]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### InputObjectParameterSet
+```
+Remove-AzureRmIotCentralApp [-PassThru] -InputObject <PSIotCentralApp> [-AsJob]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### InteractiveIotCentralParameterSet
+```
+Remove-AzureRmIotCentralApp [-PassThru] [-ResourceGroupName] <String> [-Name] <String> [-AsJob]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Deletes an existing IoT Central Application.
+
+## EXAMPLES
+
+### Example 1 Delete and IoT Central Application
+```powershell
+PS C:\> Remove-AzureRmIotCentralApp -ResourceGroupName "MyResourceGroupName" -Name "MyAppResourceName"
+```
+
+Deletes the provided IoT Central Application.
+
+## PARAMETERS
+
+### -AsJob
+Run cmdlet as a job in the background.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -InputObject
+Iot Central Application Input Object.
+
+```yaml
+Type: PSIotCentralApp
+Parameter Sets: InputObjectParameterSet
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByValue)
+Accept wildcard characters: False
+```
+
+### -Name
+Name of the Iot Central Application Resource.
+
+```yaml
+Type: String
+Parameter Sets: InteractiveIotCentralParameterSet
+Aliases:
+
+Required: True
+Position: 1
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -PassThru
+{{Fill PassThru Description}}
```

Comment:
Should not have any empty descriptions in markdown files

Reply:
Done